ui/app/containers/events.spec.tsx

```diff
@@ -27,14 +27,14 @@ describe("<EventList>", function() {
   });
 
   describe("refresh", function() {
-    it("refreshes events when mounted.", function () {
+    it("refreshes events when mounted.", () => {
```

Comment:
You didn't replace all the `describe` functions.
Why don't they need a proper closures anymore?